File: 3. bolum/while/php/sonsuz.php
<?php


function main()
{

    $k = 1;
    $toplam = 0;
    while (1) {
        $sayi = readline("Sayı giriniz : ");
        if ($sayi == -1)
            break;
        $k++;
    }
    $ortalama = $toplam / $k;
    printf("%d sayisini ortalamasi = %f \n",$k,$ortalama);
}

main();

File: 3. bolum/while/python/sonsuz.py
def main():
    k = 1
    toplam = 0
    while (1):
        sayi = input("Sayi giriniz")

        if int(sayi) == -1:
            break
        k += 1
    ortalama = toplam / k
    print("%2d sayisinin ortalamasi = %2d \n" % (k, ortalama))


main()
